### DISCUSSION TITLE
Authorization lua script

### DISCUSSION THREAD

**Discussion by amarsharma222** [vernemq/vernemq#2163] [General]
Hi VerneMQ team , 

Looking for help with any documentation link to manage the access control to topics using lua script. will be very helpful. 

Thanks
Amar

**Comment by ioolkos** [comment]
@amarsharma222 Lua (vmq_diversity) enables authentication and authorization for databases. 
The documentation on this is here: https://docs.vernemq.com/configuring-vernemq/db-auth
We should be able to help, given specific additional questions.
